Source/Renderer/AutomaticUniforms.js

```diff
@@ -1478,6 +1478,25 @@ define([
             }
         }),
 
+         /**
+         * An automatic GLSL uniform representing the splitter position to use when rendering imagery layers with a splitter.
+         *
+         * @alias czm_splitPosition
+         * @glslUniform
+         *
+         *
+         * @example
+         * // GLSL declaration
+         * uniform float czm_splitPosition;
+         */
+        czm_splitPosition : new AutomaticUniform({
```

Comment:
Also throughout, I think this would benefit from a more precise name.  In the context of the Cesium renderer, "split position" is too general.  Perhaps `imagerySplitPosition`?